ec2_metadata.py

```diff
@@ -6,14 +6,15 @@
 
 __author__ = 'Adam Johnson'
 __email__ = 'me@adamj.eu'
-__version__ = '1.0.0'
+__version__ = '1.1.0'
 
 __all__ = ('ec2_metadata',)
 
 
-SERVICE_URL = 'http://169.254.169.254/2016-09-02/'
+SERVICE_URL = 'http://169.254.169.254/latest/'
```

Comment:
there's a reason it's fixed to a particular version, to isolate users from any possible backwards incompatible changes that AWS might make. this will be upgraded for each release they do after verifying the results are all the same. Please revert